pilot/pkg/serviceregistry/kube/controller/ambient/ambientindex.go

```diff
@@ -492,7 +495,7 @@ func (a *index) NetworksSynced() {
 	a.networkUpdateTrigger.MarkSynced()
 }
 
-func (a *index) Run(stop <-chan struct{}) {
+func (a *index) RunStatus(stop <-chan struct{}) {
```

Comment:
**nit**: with this renamed it makes a.queue look off. Should it be `statusQueue` instead?